CRAMv2.1.tex

```diff
@@ -1214,6 +1216,26 @@ \subsubsection*{General notes}
 container headers, namely sequence id, alignment start and span, container start 
 byte offset and slice byte offset inside the container. 
 
+The currently supported index formats are listed in table \ref{tab:index}.
```

Comment:
The table arrives at a silly place, so maybe we can coerce it to appear after the Indexing chapter heading.

Also this needs to be applied to CRAMv3.tex too.

The description itself is fine.  Thanks.  Htslib doesn't actually support .cram.bai though, but I think it's fine to list the options in the spec and having .crai first is good.